### PR TITLE
[service.subtitles.subtituloses] and [service.subtitles.tusubtitulo] broken

### PR DESCRIPTION
**Pull request by josecurioso**
### Description
Both addons are not maintained anymore since both webpages either changed or went down so none of them work anymore.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@josecurioso please also bump the version number for both

**Comment by josecurioso** [comment]
Done

**Comment by josecurioso** [comment]
[service.subtitles.subtituloses] is now v0.0.7 and [service.subtitles.tusubtitulo] is v1.1.0

File: service.subtitles.subtituloses/addon.xml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="service.subtitles.subtituloses"
        name="Subtitulos.es"
-       version="0.0.5"
+       version="0.0.6"
        provider-name="m0r3">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
   </requires>
+  <broken>deprecated</broken>
```
Comment (Rechi) [inline service.subtitles.subtituloses/addon.xml]: `broken` tag has to go inside `<extension point="(kodi|xbmc).addon.metadata">`

File: service.subtitles.tusubtitulo/addon.xml [hunk 2]
```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="service.subtitles.tusubtitulo"
        name="TuSubtitulo"
-       version="1.0.0"
+       version="1.1.0"
        provider-name="El_Happy">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
   </requires>
+  <broken>deprecated</broken>
```
Comment (Rechi) [inline service.subtitles.tusubtitulo/addon.xml]: `broken` tag has to go inside `<extension point="(kodi|xbmc).addon.metadata">`

**Pull request merged**